packages/shared/src/components/auth/AuthenticationBanner.tsx

```diff
@@ -1,29 +1,44 @@
-import React, { ReactElement } from 'react';
+import React, { CSSProperties, ReactElement } from 'react';
+import classNames from 'classnames';
 import classed from '../../lib/classed';
 import { OnboardingHeadline } from './OnboardingHeadline';
 import AuthOptions, { AuthDisplay } from './AuthOptions';
 import { AuthTriggers } from '../../lib/auth';
 import { MemberAlready } from '../onboarding/MemberAlready';
 import { useAuthContext } from '../../contexts/AuthContext';
+import { useViewSize, ViewSize } from '../../hooks';
+import LoginButton from '../LoginButton';
+import { bottomBannerBaseClasses, BottomBannerContainer } from '../banners';
 
-const Container = classed('div', 'flex flex-col w-[23.25rem]');
+const Section = classed('div', 'flex flex-col w-[23.25rem]');
+const style: CSSProperties = {
+  background: `linear-gradient(270deg, rgba(206, 61, 243, 0.16) 0%, rgba(113, 71, 237, 0.16) 100%), var(--theme-background-primary)`,
+};
 
 export function AuthenticationBanner(): ReactElement {
   const { showLogin } = useAuthContext();
+  const isLaptop = useViewSize(ViewSize.Laptop);
+
+  if (!isLaptop) {
```

Comment:
Hm I thought we would be having this condition below inside `<BottomBannerContainer className="gap-6 py-10" style={style}>` where we either show the sections or the login buttons. Feels more natural to me. 
But not saying this is bad either, just putting it out there.

Reply:
The `LoginButton` has its own container already, which we will have to style anyway (gap, padding, etc.) - also, just not a fan of ternary operators for large renders.